apps/web/vibes/soul/sections/reviews/index.tsx

```diff
@@ -110,3 +153,11 @@ export function ReviewsSkeleton() {
     </StickySidebarLayout>
   )
 }
+
+export function Reviews({ reviewsLabel = 'Reviews', ...props }: Props) {
+  return (
+    <Suspense fallback={<ReviewsSkeleton />}>
+      <ReviewsImpl {...props} reviewsLabel={reviewsLabel} />
+    </Suspense>
+  )
+}
```

Comment:
This is only needed because I resolve reviews to know if I want to render empty component or not. See above.